runner/src/test/java/com/redhat/service/bridge/runner/it/End2EndTestIT.java

```diff
@@ -57,6 +58,7 @@
 @QuarkusTestResource(KafkaResource.class)
 @QuarkusTestResource(PostgresResource.class)
 @QuarkusTestResource(KeycloakResource.class)
+@Disabled("The shard has to be deployed to a kubernetes cluster in order to make this e2e test working")
 public class End2EndTestIT {
 
```

Comment:
Disabled e2e tests

Reply:
Do we have a ticket to link about enabling/create new e2e test?

Reply:
https://issues.redhat.com/browse/MGDOBR-150